roles/pulp_database/vars/Fedora.yml

```diff
@@ -1 +1,2 @@
 ---
+postgresql_python_library: python3-psycopg2
```

Comment:
It looks like this is unnecessary:
https://github.com/geerlingguy/ansible-role-postgresql/search?q=psycopg2